packages/tools-iottwinmaker/functional-tests/basic-functional/basic-functional-constants.ts

```diff
@@ -0,0 +1,56 @@
+import * as path from 'path';
+import * as os from 'os';
+
+export const localResourcesDir = path.join(__dirname, 'basic-functional-resources');
+export const timestamp = Date.now() % 10000;
+export const workspaceId = `tmdk-functional-test-workspace-${timestamp}`;
+export const region = 'us-east-1';
+export const tmdkDirectory = path.join(os.tmpdir(), 'functional-tmdk-test');
+export const tmdkFile = 'tmdk.json';
+export const entitiesFile = 'entities.json';
+
+export const componentType1Input = {
```

Comment:
need to add types to the entity definitions and inputs

Reply:
Will do